operator/v1/types.go

```diff
@@ -40,6 +41,14 @@ type ClusterManagerSpec struct {
 	// +optional
 	// +kubebuilder:default=quay.io/open-cluster-management/placement
 	PlacementImagePullSpec string `json:"placementImagePullSpec,omitempty"`
+
+	// NodeSelector defines which Nodes the Pods are scheduled on. The default is an empty list.
+	// +optional
+	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+
+	// Tolerations defines the tolerations for the Pods. The default is an empty list.
```

Comment:
I think you would need to ref this to the definition of toleration. Current doc is not clear.

Reply:
done. put them in NodePlacement

Reply:
done